eng/WpfArcadeSdk/tools/WpfProjectReference.targets

```diff
@@ -54,19 +54,19 @@
     <WpfProjectPath Include="UIAutomationClientSideProviders" ProjectPath="$(WpfSourceDir)UIAutomation\UIAutomationClientSideProviders\UIAutomationClientSideProviders.csproj" />
 
 
-    <WpfProjectPath Include="PresentationNative_cor3" ProjectPath="$(WpfSourceDir)PresentationNative\dll\PresentationNative.vcxproj">
+    <WpfProjectPath Include="PresentationNative_net6" ProjectPath="$(WpfSourceDir)PresentationNative\dll\PresentationNative.vcxproj">
```

Comment:
Shouldn't these includes use the suffix property to build the name?

Reply:
Thanks, @batzen.